include/exadg/operators/inverse_mass_operator.h

```diff
@@ -266,38 +266,41 @@ class InverseMassOperator
   void
   apply(VectorType & dst, VectorType const & src) const
   {
+    // Note that the inverse mass operator might be called like inverse_mass.apply(dst, dst),
```

Comment:
I moved this out to have it for both variants.
One of the original `InverseOperatorMassHdiv` or `InverseOperatorMass` before the merge in #749 had this.
For safety, allow it for the merge one as well.
The test below actually checks the case `&dst == &src` and the regular one.

Reply:
I first did not do it and found that the test as done below was failing.
Then I moved it and saw that for the `InverseMassType::MatrixfreeOperator` it was also needed for the case `$src==$dst`. But maybe this is just a side-effect of the other problem we have, I will test this again before we have a chat.